src/ScClient.ts

```diff
@@ -122,21 +128,31 @@ export class ScClient {
     sendData();
   }
 
+  private formSendAnswer(resolve: (arg: any) => void, reject: (arg: any) => void, success: any, errors?: any) {
```

Comment:
It's probably better to name just `sendAnswer`

Reply:
This name I agreed with @aleksgrin. What do you think about this proposal, @aleksgrin

Reply:
I thought of naming it this way but to my mind this not what function does. Send sounds like success, but this function can resolve or reject. 

This one is not ideal too... The best a can think of is to put info about error handling. Maybe `resolveOrReject()` or something like that.

What do you think?

Reply:
Agree, let's confirm `resolveOrReject()`